core/src/traits.rs

```diff
@@ -74,3 +81,45 @@ pub trait Func<Input> {
     /// use cases.
     fn call(i: Input) -> Self::Output;
 }
+
+/// This is a user-implementable alternative to `FnMut`.
+///
+/// Not necessary if/when the FnMut trait is implementable in stable Rust.
+///
+/// ```
+/// # #[macro_use]
+/// # extern crate frunk;
+/// # use frunk::{FuncMut, PolyMut};
```

Comment:
Because some items are exported under multiple paths, I've been making it a point for doc examples to always show the intended paths for using stuff from `frunk`, so unhide this.

Likewise for the `#[macro_use] extern crate`